### PR TITLE
RIP one letter vars

### PR DESCRIPTION
**Pull request by Diegoflores31**
# About the pull request

changes every single one letter var from /mutators (AKA strains folder) making all of them having a relatively uniform name so new strains created can fill up their vars using the same format hopefully.

# Explain why it's good for the game

are you happy now harry ? Merge my lurkerrr 😢 

# Testing Photographs and Procedure

<details>

image

<summary>Screenshots & Videos</summary>

</details>

# Changelog

:cl:
code: refactored strain mutator code var names
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/boiler/trapper.dm [hunk 1]
```diff
@@ -64,30 +64,30 @@
...
 		return
 
-	var/mob/living/carbon/human/H = A
+	var/mob/living/carbon/human/target_human = target_atom
 	var/datum/effects/boiler_trap/found = null
-	for (var/datum/effects/boiler_trap/F in H.effects_list)
+	for (var/datum/effects/boiler_trap/F in target_human.effects_list)
 		if (F.cause_data?.resolve_mob() == bound_xeno)
 			found = F
 			break
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/mutators/strains/boiler/trapper.dm:83]: 👀 what's all this then

Comment (Diegoflores31) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/boiler/trapper.dm:83]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/praetorian/dancer.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /datum/xeno_mutator/praetorian_dancer
 	// My name is Cuban Pete, I'm the King of the Rumba Beat
 	name = "STRAIN: Praetorian - Dancer"
-	description = "You are now a paragon of agility. You lose the ability to spit and lose some armor. You gain an ability that lets you dodge through tallhosts (cancelled on slash) and two powerful tail abilities each enhanced by a tag placed on your opponents via slashing."
+	description = "You are now a paragon of agility. You lose the ability to spit and lose some armor. You gain an ability that lets you dodge through tallhosts (cancelled on slash) and two powerful tail abilities each enhanced by a target_tag placed on your opponents via slashing."
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/mutators/strains/praetorian/dancer.dm:4]: 👀

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/praetorian/dancer.dm (hunk 2 above)
Comment (Diegoflores31) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/praetorian/dancer.dm:4]: 💀

**Comment by fira** [comment]
wow

**Pull request merged**